File: bankingtool.client/src/app/dto/response/token-response.ts
import { ClaimDto } from "../claim-dto";
import { GetActionsByUserIdDto } from "../get-actions-by-user-id-dto";

export interface TokenResponse {
  accessToken: string;
  claims: ClaimDto[];
  expireIn: string;
  actionPaths: GetActionsByUserIdDto[];
}

File: BankingTool.Model/Dto/ResponseDto/TokenResponse.cs
namespace BankingTool.Model.Dto.Response
{
    public class TokenResponse
    {
        public string AccessToken { get; set; }
        public List<ClaimDto> Claims { get; set; }
        public string ExpireIn { get; set; }
        public List<GetActionsByUserIdDto> ActionPaths { get; set; }

    }
}
